src/Symfony/Component/PropertyInfo/Type.php

```diff
@@ -43,6 +44,7 @@ class Type
         self::BUILTIN_TYPE_ARRAY,
         self::BUILTIN_TYPE_CALLABLE,
         self::BUILTIN_TYPE_NULL,
+        self::BUILTIN_TYPE_ITERABLE,
```

Comment:
phpdoc of `getBuiltinType()` should be updated too :)

Reply:
good catch, done

Reply:
@dunglas Can you push your fix?

Reply:
oops, done